modules/installation/proc-rhdh-deploy-eks-operator.adoc

```diff
@@ -18,7 +18,7 @@ You can install the {product-short} Operator on {eks-short} using the https://ol
 * You have set the context to the {eks-short} cluster in your current `kubeconfig`. For more information, see https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html[Creating or updating a kubeconfig file for an Amazon {eks-short} cluster].
 * You have installed `kubectl`. For more information, see https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html[Installing or updating kubectl].
 * You have subscribed to `registry.redhat.io`. For more information, see https://access.redhat.com/RegistryAuthentication[{company-name} Container Registry Authentication].
-* You have installed the Operator Lifecycle Manager (OLM). For more information about installation and troubleshooting, see https://operatorhub.io/how-to-install-an-operator#How-do-I-get-Operator-Lifecycle-Manager?[How do I get Operator Lifecycle Manager?]
+* You have installed the Operator Lifecycle Manager (OLM). For more information about installation and troubleshooting, see https://olm.operatorframework.io/docs/getting-started/[OLM QuickStart] or https://operatorhub.io/how-to-install-an-operator#How-do-I-get-Operator-Lifecycle-Manager?[How do I get Operator Lifecycle Manager?]
```

Comment:
@nickboldt According to the [RHIDP-2707](https://issues.redhat.com/browse/RHIDP-2707) description, "the operator install section doesn't mention that on Kubernetes clusters, you cannot simply discover the RHDH operator." Should we consider adding a note about Kubernetes clusters in this section?

Reply:
Yes, once OLM is installed and the RH CatalogSource is added, I think we can list all the available operators from that source and easily discover the RHDH one with this command:
```sh
kubectl get packagemanifests -n ${CatalogSourceNamespace}
```

For example:
```sh
$ k get packagemanifests -n rhdh-operator | grep rhdh      
rhdh                                           Red Hat Operators     8m21s
```

@hmanwani-rh Do you want this to be added to this page as well?

IMO, this is quite "simple" enough to discover the RHDH operator on non-OCP clusters, even without a UI. But do we need to mention this here? IMO, this is documenting how to install the operator, so we already know what to expect (like the name of the CatalogSource, namespace, ...).
That said, verifying that the RHDH operator shows up in the PackageManifests might still be helpful for troubleshooting.